scripts/update_composite_view.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python3
+
+from decouple import config
+import requests
+import json
+from typing import Dict
+from typing import List
+from urllib import parse
+import sys
+import os
+
+# https://towardsdatascience.com/understanding-python-imports-init-py-and-pythonpath-once-and-for-all-4c5249ab6355
+# TODO: Use PYTHONPATH instead
+PROJECT_ROOT = os.path.abspath(os.path.join(
+                  os.path.dirname(__file__),
+                  os.pardir)
+)
+sys.path.append(PROJECT_ROOT)
+
+from utils import absolute_from_rel_file_path
+
+# TOKEN has to be set
+# in file .env (project root): TOKEN="..."
+TOKEN = config('TOKEN')
+NEXUS_ENVIRONMENT = config('NEXUS')
+ORG = config('ORG')
+PROJECT = config('PROJECT')
+VERIFY_SSL: bool = bool(int(config('VERIFY_SSL'))) # throws an uncaught error if not numerical / integer
+
+def get_composite_view(id: str) -> Dict:
+    """
+    Given the id of a composite view, fetches it from Nexus.
+
+    :param id: The composite view's id, e.g, connectome-projection-composite-01.
+    :return: The composite view fetched from Nexus.
+    """
+    req = requests.get(
+        NEXUS_ENVIRONMENT + '/views/' + ORG + '/' + PROJECT + '/' + id,
+        headers={
+            'Content-Type': 'application/json',
+            'Authorization': f'Bearer {TOKEN}'
+        },
+        verify=VERIFY_SSL
+    )
+
+    return req.json()
+
+def update_composite_view(composite_view: Dict) -> None:
```

Comment:
Ich würde hier nur 1 Script machen welches Register und Update in einem macht (compositeview immer zuerst holen). Oder Scripts etwas verallgemeinern. 
Org/Project wäre evtl. besser als arguments zu haben und nicht im Environment zu haben, weil sich diese oft ändern können resp. bei lokal/remote Zugriff evtl. etwas umständlich in der Handhabung.

Reply:
I'll try to handle all in one composite view with multiple ES projections, see https://github.com/Connectome-Consortium/rescs_shacl_shapes/issues/17